apps/vpnmanager/scripts/processGsheet.js

```diff
@@ -0,0 +1,12 @@
+async function main() {
+  const { NEXT_PUBLIC_APP_URL, SECRET_TOKEN } = process.env;
```

Comment:
May be it works (as this script is probably called outside of Next.js), but when it comes to Next.js apps it's best not to [destructure](https://nextjs.org/docs/app/api-reference/next-config-js/env) process.env:
>Next.js will replace process.env.customKey with 'my-value' at build time. Trying to destructure process.env variables won't work due to the nature of webpack [DefinePlugin](https://webpack.js.org/plugins/define-plugin/).